src/main/java/com/redhat/devtools/intellij/common/utils/ToolsConfigHelper.java

```diff
@@ -0,0 +1,33 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v20.html
+ *
+ * Contributors:
+ * Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package com.redhat.devtools.intellij.common.utils;
```

Comment:
separated methods that deal with the tools config from the methods that deal with the kubernernetes-client config. -> created a new class `ToolsConfigHelper`